bootstrap-demo/kubernetes-dns/kubernetes/akka-cluster.yml

```diff
@@ -0,0 +1,89 @@
+---
+apiVersion: "apps/v1beta2"
+kind: Deployment
+metadata:
+  name: "bootstrap-demo-kubernetes-dns"
+  labels:
+    appName: "bootstrap-demo-kubernetes-dns"
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      appName: "bootstrap-demo-kubernetes-dns"
+#update
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+    type: RollingUpdate
+ #update
+
+  template:
+    metadata:
+      labels:
+        appName: "bootstrap-demo-kubernetes-dns"
+    spec:
+      restartPolicy: Always
+      containers:
+      - name: "bootstrap-demo-kubernetes-dns"
+        image: "bootstrap-demo-kubernetes-dns:1.3.3.7"
+        imagePullPolicy: IfNotPresent
+        readinessProbe:
+          httpGet:
+            path: /ready
+            port: 8080
+        livenessProbe:
+          httpGet:
+            path: /alive
+            port: 8080
+        env:
+        ports:
+        - containerPort: 8558
+          name: management
+        - containerPort: 2552
+          name: remoting
+        volumeMounts: []
+      volumes: []
+---
+#headless
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    appName: "bootstrap-demo-kubernetes-dns"
+  annotations:
+    service.alpha.kubernetes.io/tolerate-unready-endpoints: "true"
```

Comment:
It looks like this annotation will be removed in Kubernetes 1.13 (https://github.com/kubernetes/kubernetes/issues/63741) to be replaced by `Service.spec.publishNotReadyAddresses` (https://github.com/kubernetes/kubernetes/commit/944e07480fa4066fb1345cd1a81d74b3bbb78440). As a general rule, I don't think we should be telling people to use Alpha K8s features. Should we wait until this is sorted out in K8s before releasing?

Reply:
In the docs at https://github.com/akka/akka-management/pull/278/files#diff-e737933b9744b8db3a71650e02a1e627R50 we recommend adding both to avoid problems with different versions/assumptions - that seems reasonably safe?

Reply:
I don't think we know; would have to be looked into. I'll mark it as an action item though.
@chbatey is now laser focused on shipping another thing for a customer, we'll get his attention later on to this.